inspirehep/modules/theme/static/scss/format/brief-results.scss

```diff
@@ -508,4 +508,17 @@ i.fa-chevron-right.facet-slider {
 
 .brief-results-sortby {
 	color: #737373;
+}
+
+div > .modal-dialog {
+    width: 900px;
+}
+
+#textarea-cite {
+    resize: none;
+    width: 868px;
+    min-height: 237px;
+    background-color: #f5f5f5;
```

Comment:
this is the grey that was used in the <pre> tag that now is a <textarea>. I don't think we are using it anywhere else. Should I keep it like this?